ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/server/resources/KsqlResource.java

```diff
@@ -232,14 +241,20 @@ public EndpointResponse handleKsqlStatements(
           request,
           distributedCmdResponseTimeout);
 
-      final KsqlRequestConfig requestConfig =
-          new KsqlRequestConfig(request.getRequestProperties());
+      final Map<String, Object> requestProperties = request.getRequestProperties();
+      denyListPropertyValidator.validateAll(requestProperties);
```

Comment:
I don't think it makes sense to validate request properties.  I think this may just break stuff.  @vpapavas thoughts?

Reply:
Removed this validation. I asked @vpapavas and she said we shouldn't do it.